codespell_lib/data/dictionary.txt

```diff
@@ -21406,6 +21406,10 @@ interpolaion->interpolation
 interpolaiton->interpolation
 interpolar->interpolator
 interpolayed->interpolated
+interpoloate->interpolate
+interpoloated->interpolated
```

Comment:
For verbs, we usually suggest adding all possible forms:
```
interpoloates->interpolates
interpoloating->interpolating
```

Reply:
Thanks for the review. I updated this PR.